src/SprykerEco/Shared/AmazonQuicksight/AmazonQuicksightConstants.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Shared\AmazonQuicksight;
+
+/**
+ * Declares global environment configuration keys. Do not use it for other class constants.
+ */
+interface AmazonQuicksightConstants
+{
+    /**
+     * Specification:
+     * - The ID for the AWS account that contains your Amazon QuickSight account.
+     *
+     * @api
+     *
+     * @var string
+     */
+    public const AWS_ACCOUNT_ID = 'QUICKSIGHT:AWS_ACCOUNT_ID';
```

Comment:
```suggestion
    public const AWS_ACCOUNT_ID = 'AMAZON_QUICKSIGHT:AWS_ACCOUNT_ID';
```
please check everywhere